tests/PhpPresentation/Tests/Writer/PowerPoint2007/PptChartsTest.php

```diff
@@ -107,20 +107,26 @@ public function testChartIncludeSpreadsheet(): void
         $this->assertZipXmlElementExists('ppt/charts/' . $oShape->getIndexedFilename(), $element);
         $element = '/c:chartSpace/c:chart/c:plotArea/c:lineChart/c:ser';
         $this->assertZipXmlElementExists('ppt/charts/' . $oShape->getIndexedFilename(), $element);
+        $this->assertZipXmlElementCount('ppt/charts/' . $oShape->getIndexedFilename(), $element, 1);
```

Comment:
Could you for `testChartIncludeSpreadsheet` a `dataProvider` which pass in parameter 
- each chartType (Line/Radar....) where you have made a change in `PptCharts`
- a string for element xml `c:lineChart` relative to each chartType
?

That allows me to check that each change in `PptCharts` is ok.